### ISSUE TITLE
Docker image

### ISSUE THREAD

**Issue by Prabhusarathy18** [wenmin-wu/jupyter-tabnine#59]
Please check for Dockerized build.  Facing multiple issues while creating image and also while docker run.

**Comment by wenmin-wu** [comment]
Please refer to https://github.com/wenmin-wu/jupyter-tabnine/blob/master/README.md#build-server-image, do let me know if you have any other problems.

**Comment by Prabhusarathy18** [comment]
Thanks for response.  Now the image build is very fine.  I'm getting CORS issues while specifying remote_server_url in the config file.  Is there any fix for this?

**Comment by wenmin-wu** [comment]
Did you modify the code? The CORS issue had been solved here https://github.com/wenmin-wu/jupyter-tabnine/blob/master/go/cmd/server.go#L33

**Comment by Prabhusarathy18** [comment]
@wenmin-wu  Yes it is working now.  But Im not getting any results now.  I pointed the remote-url param as http://0.0.0.0:9999/.  Also tried by hard-coding inside main.js file.  To my understanding the ajax call is loading too long.  May be network issue.  Please check this once.

Also the image was taken from docker-hub.  Also tried with build-script.  Both fails for me.

**Comment by Prabhusarathy18** [comment]
Also tried with postman,
http://0.0.0.0:8888/tabnine?data=%7B%22version%22%3A%221.0.7%22%2C%22request%22%3A%7B%22Autocomplete%22%3A%7B%22filename%22%3A%22%2FUntitled2044.py%22%2C%22before%22%3A%22i%22%2C%22after%22%3A%22%22%2C%22region_includes_beginning%22%3Atrue%2C%22region_includes_end%22%3Atrue%2C%22max_num_results%22%3A10%7D%7D%7D

It is not fetching any results.

**Comment by wenmin-wu** [comment]
Hi @Prabhusarathy18, it's caused by the TabNine binary refer to #60, could you please enter your container and check the TabNine version? I solved this by downloading a fixed version from https://github.com/codota/tabnine-vim/tree/master/binaries/2.9.2

**Comment by Prabhusarathy18** [comment]
Using 2.9.2 version.  Still facing same issue.

**Comment by Prabhusarathy18** [comment]
@wenmin-wu  I think server inside the container is stopping.  At first the container memory is 100+MB but after 10-15 sec it is under 10MB.  So it is understood that some process inside the container is killed after 10-15sec.  Also I checked with latest version of TabNine 3.2.28  which was specified #60.  Please check this.
